src/utils.js

```diff
@@ -270,7 +272,21 @@ function escapeLocalIdent(localident) {
     localident
       // For `[hash]` placeholder
       .replace(/^((-?[0-9])|--)/, "_$1")
-      .replace(filenameReservedRegex, "-")
+      .replace(filenameReservedRegex, (_, $1) => {
+        // normalize Windows path `\` as unix `/` for constancy
```

Comment:
I'm still not sure how to distinguish `path.sep` vs css `\\` here.

Reply:
hm, I think we can ignore such case, do you see any critical places here?

Reply:
> hm, I think we can ignore such case, do you see any critical places here?

Nope, the test just fails before current changes here.